scripts/image/setup_full_no_backports.sh

```diff
@@ -0,0 +1,224 @@
+    fi
+done
+
+
+
+# Ensure /etc/rc.local exists and is executable
+if [ ! -f /etc/rc.local ]; then
+    echo "#!/bin/sh -e" > /etc/rc.local
+    echo "exit 0" >> /etc/rc.local
+fi
+chmod +x /etc/rc.local
+
+# Remove 'exit 0' from /etc/rc.local if it exists
+sed -i '/^exit 0$/d' /etc/rc.local
+
+# Append the commands
+cat <<EOF >> /etc/rc.local
+
+# Commands to install Overwitch udev rules
+cd $OVERWITCH_SOURCE_PATH/$OVERWITCH_FOLDER_NAME/udev
+make install
+
+exit 0
+EOF
+
+echo "Installing Pushpin"
+cd $PUSHPIN_SOURCE_PATH
+git clone https://github.com/kingajanicka/pushpin-groovebox $PUSHPIN_SOURCE_PATH/pushpin-groovebox
+ln -s $PUSHPIN_SOURCE_PATH/pushpin-groovebox/pushpin_groovebox.conf $PIPEWIRE_CONFIGURATION_PATH/pushpin_groovebox.conf
+
+echo "Installing Pushpin deps"
+python -m venv $PYTHON_ENVIRONMENT_PATH/venv_pushpin_groovebox
+
+. $PYTHON_ENVIRONMENT_PATH/venv_pushpin_groovebox/bin/activate
+cd $PUSHPIN_SOURCE_PATH/pushpin-groovebox
+
+pip install -r requirements.txt
+pip install ratelimit
+pip install pytest
+pip install pytest-asyncio
+
+chown -R $USER_NAME:audio $USER_HOME_PATH 
+chmod -R 755 $USER_HOME_PATH
+
+
+echo "Configuring /boot/firmware/config.txt for HiFiBerry DAC8x"
+
+CONFIG_FILE="/boot/firmware/config.txt"
+
+# Function to safely comment or replace lines in the file
+update_config_file() {
+    local file=$1
+    local match=$2
+    local replace=$3
+
+    # If the match exists, replace it; otherwise, append the replacement
+    if grep -q "^${match}" "$file"; then
+        sed -i "s|^${match}.*|${replace}|g" "$file"
+    else
+        echo "$replace" >> "$file"
+    fi
+}
+
+# Ensure the file exists
+if [ ! -f "$CONFIG_FILE" ]; then
+    echo "Error: $CONFIG_FILE does not exist. Exiting."
+    exit 1
+fi
+
+# Remove or disable onboard audio
+sed -i '/^dtparam=audio=on/d' "$CONFIG_FILE"
+update_config_file "$CONFIG_FILE" "dtoverlay=vc4-kms-v3d" "dtoverlay=vc4-kms-v3d,noaudio"
+
+# Add HiFiBerry DAC8x overlay
+update_config_file "$CONFIG_FILE" "dtoverlay=hifiberry-dac8x" "dtoverlay=hifiberry-dac8x"
+echo "Configuration of /boot/firmware/config.txt completed."
+
+echo "Configuring /etc/security/limits.conf for audio settings"
+
+LIMITS_CONF_FILE="/etc/security/limits.conf"
+
+# Ensure the file exists
+if [ ! -f "$LIMITS_CONF_FILE" ]; then
+    echo "Error: $LIMITS_CONF_FILE does not exist. Exiting."
+    exit 1
+fi
+
+# Add or update audio settings
+update_limits_conf() {
+    local file=$1
+    local match=$2
+    local replace=$3
+
+    # If the match exists, replace it; otherwise, append the replacement
+    if grep -q "^${match}" "$file"; then
+        sed -i "s|^${match}.*|${replace}|g" "$file"
+    else
+        echo "$replace" >> "$file"
+    fi
+}
+
+update_limits_conf "$LIMITS_CONF_FILE" "@audio - rtprio" "@audio - rtprio 95"
+update_limits_conf "$LIMITS_CONF_FILE" "@audio - memlock" "@audio - memlock unlimited"
+
+echo "Configuration of /etc/security/limits.conf completed."
+
```

Comment:
No idea about any of this due to not having a Hifiberry but that looks cool as heck. 👀